extensions/chat/common/client.mjs

```diff
@@ -95,26 +95,46 @@ Usage: !/<Command> <arg1> <arg2> ...
         document.getElementById('message_history').innerHTML += '<span>Invalid Command</span><br>';
         this.listCommand();
       }
-    };
+    }
   }
 
+  /**
+   * Encode the html in case of XSS
+   * @HTMLEncode
+   * @param {string} str - the string being encoded
+   */
   HTMLEncode(str) {
     return $('<div/>').text(str).html();
   }
 
+  /**
+   * Get a private message from other player
+   * @s2c_getPrivateMessage
+   * @param {object} arg - Information of the private message
+   */
   s2c_getPrivateMessage(arg) {
     document.getElementById('message_history').innerHTML += '<span>Private Message From ' + this.HTMLEncode(arg.msg_from) + ': ' + this.HTMLEncode(arg.msg) + '</span><br>';
   }
 
+  /**
+   * Display a private message after sending the private message to other player
+   * @s2c_senderPrivateMessage
+   * @param {object} arg - Information of the private message
+   */
   s2c_sendedPrivateMessage(arg) {
     document.getElementById('message_history').innerHTML += '<span>Private Message To ' + this.HTMLEncode(arg.msg_to) + ': ' + this.HTMLEncode(arg.msg) + '</span><br>';
   }
 
+  /**
+   * Get a Broadcast message
+   * @onExtensionBroadcast
+   * @param {object} arg - Information of the broadcast message
+  */
   onExtensionBroadcast(arg) {
     arg.msg_from = $('<div/>').text(arg.msg_from).html();
     arg.msg = $('<div/>').text(arg.msg).html();
     document.getElementById('message_history').innerHTML += '<span>' + arg.msg_from + ': ' + arg.msg + '</span><br>';
   }
-};
+}
```

Comment:
Same as above, I'm not sure.

Reply:
sorry for no PR description
all fixed by eslint --fix